IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ElectricalDeviceCommon/DocPropertySet.xml

```diff
@@ -19,6 +19,7 @@
 		<DocProperty xsi:nil="true" href="Power_13VYb5tj10cOr5hZsTbt3R" />
 		<DocProperty xsi:nil="true" href="NominalPowerConsumption_0PrrQ0qSaHuO00025QrE$V" />
 		<DocProperty xsi:nil="true" href="NumberOfPowerSupplyPorts_2zCP6ydR933h5ygkKmdVFt" />
+		<DocProperty xsi:nil="true" href="FireRating_0eTJm0qT0HuO00025QrE$V" />
```

Comment:
Would have never have guessed that electrical devices also have a fire rating!

Reply:
It was really important (ask Florent).